src/node_env_var.cc

```diff
@@ -175,10 +175,7 @@ Local<Array> RealEnvStore::Enumerate(Isolate* isolate) const {
   int env_v_index = 0;
   for (int i = 0; i < count; i++) {
 #ifdef _WIN32
-    // If the key starts with '=' it is a hidden environment variable.
```

Comment:
I’m not sure why this line is being removed, though?

Reply:
Whoops, I'll add that back in. Thanks for reviewing. :)

Reply:
done